packages/lib/bookings/findQualifiedHosts.ts

```diff
@@ -0,0 +1,43 @@
+import type { Host } from "@calcom/prisma/client";
```

Comment:
It would be so nice to not have any reference to prisma in these types of files.